ruby/ql/lib/codeql/ruby/frameworks/ActiveRecord.qll

```diff
@@ -200,7 +200,13 @@ private predicate sqlFragmentArgumentInner(DataFlow::CallNode call, DataFlow::No
   call = activeRecordQueryBuilderCall("annotate") and
   sink = call.getArgument(_)
   or
-  call = activeRecordConnectionInstance().getAMethodCall("execute") and
+  call =
+    activeRecordConnectionInstance()
+        .getAMethodCall([
```

Comment:
We can add [`create`](https://api.rubyonrails.org/classes/ActiveRecord/ConnectionAdapters/DatabaseStatements.html#method-i-create) (an alias for `insert`) to this.

I think we can also remove `select_update`? I can't find this as a query execution method.